docs/rules/id-length.md

```diff
@@ -0,0 +1,84 @@
+# Limit minimum and maximum length for identifiers (id-length)
+
+Identifier names such as `e`, `x`, `_t` or sometimes very long ones like `hashGeneratorResultOutputContainerObject` which usually makes the code harder to read and less maintainable. To prevent this, some code bases may enforce a minimum and/or maximum identifier length. (usually min 2-chars)
+
+```js
+// id-length: 1  // minimum 2-chars
```

Comment:
Ooh I see, I think this is confusing as-is. I dying think people will understand that 1 is not a length. I'd specify the options throughout.